src/components/checkboxes/template.njk

```diff
@@ -50,7 +50,16 @@
     {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}
     {%- if isConditional %} data-module="checkboxes"{% endif -%}>
     {% for item in params.items %}
-    {% set id = item.id if item.id else idPrefix + "-" + loop.index %}
+    {#- If the user explicitly sets an id, use this instead of the regular idPrefix -#}
+    {%- if item.id -%}
+      {%- set id = item.id -%}
+    {%- else -%}
+      {%- set id = idPrefix -%}
+      {#- The first id should not have a prefix so it's easy to link to from the error summary component -#}
+      {%- if not loop.first -%}
+        {%- set id = id + "-" + loop.index -%}
```

Comment:
My brain tripped over this a little bit because I think it would end up using `item.id` as the prefix, rather than `idPrefix`. Now that I've read it again, I get what's going on, but I think the multiple `{% set %}` calls combined with the very similarly named variables could be confusing.

Very much non-blocking, but it _might_ be better to do e.g.

```
{%- if loop.first -%}
  {%- set id = idPrefix %}
{% else %}
  {%- set id = id + "-" + loop.index -%}
{%- endif -%}
```

Reply:
I'll give this a go thanks for the comment